at_secondary/at_persistence_secondary_server/lib/src/event_listener/at_change_event.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:at_persistence_secondary_server/at_persistence_secondary_server.dart';
+
+///Represents the change event on the persistent key-store.
+class AtChangeEvent {
```

Comment:
we can rename to some thing specific..e.g AtPeristenceChangeEvent

Reply:
Sure Murali, Will rename to AtPersistenceChangeEvent.